src/NoFrixion.MoneyMoov/Models/Beneficiary/BeneficiaryIdentifier.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -----------------------------------------------------------------------------
+//  Filename: BeneficiaryIdentifier.cs
+// 
+//  Description: BeneficiaryIdentifier model
+// 
+//  Author(s):
+//  Saurav Maiti (saurav@nofrixion.com)
+// 
+//  History:
+//  06 11 2023  Saurav Maiti Created, Harcourt Street, Dublin, Ireland.
+// 
+//  License:
+//  Proprietary NoFrixion.
+// -----------------------------------------------------------------------------
+
+#nullable disable
+namespace NoFrixion.MoneyMoov.Models;
+
+public class BeneficiaryIdentifier
+{

```

Comment:
Should some of these props be nullable?

Reply:
Yes, Iban, account number, sort code and Bitcoin address can be null but they are all strings so nullable by default. Should I explicitly make them nullable?

Reply:
I think it's better to accurately reflect the db table. And it's clearer when making api calls. Want to avoid a situation where empty strings are saved in the db instead of nulls.

Reply:
That makes sense. I'll update it. Thanks.

Reply:
@donalnofrixion @arifmatin This model actually wasn't being used because we are exposing identifier as CounterParty in Beneficiary model. So I have removed this.